Apps/frontend/src/views/ControlPanelSide.svelte

```diff
@@ -0,0 +1,10 @@
+<script>
```

Comment:
Would prefer `ControlPanelRight`. Am also fine with the current name.

Reply:
Agree